tsconfig.json

```diff
@@ -16,11 +16,29 @@
     "noImplicitThis": true,
     "noUnusedLocals": true,
     "outDir": "./lib/",
-    "rootDir": "./src",
+    // "rootDir": "./src",
```

Comment:
Has this been accidentally commented out?

Reply:
it wasn't because we are importing from 2 roots like `src` and `cypress`